ocaml/idl/datamodel_host.ml

```diff
@@ -915,7 +948,7 @@ let host_query_ha = call ~flags:[`Session]
       ~pool_internal:true
       ~hide_from_docs:true
       ~name:"crl_install"
-      ~doc:"Install a TLS Certificate Revocation List to this host."
+      ~doc:"Install a TLS CA-issued Certificate Revocation List to this host."
```

Comment:
might be overkill, but do we need to add a `Changed` to the lifecycle when we are modifying docs?

Reply:
I think this would add noise - because as a user you would expect a different behaviour when there is none.

Reply:
I don't think so, we've modified docs before and the lifecycle was not changed.

Reply:
`Changed` is for interface changes that are in some way not backwards compatible. See https://xapi-project.github.io/xen-api/evolution.html.

So: no.